src/humancellatlas/data/metadata/__init__.py

```diff
@@ -139,8 +148,11 @@ def __init__(self, json: JSON):
         self.biological_sex = content['biological_sex']
 
     @property
-    def organism_age_in_seconds(self):
-        return AgeRange.parse(self.organism_age, self.organism_age_unit)
+    def organism_age_in_seconds(self) -> Optional[AgeRange]:
+        if self.organism_age and self.organism_age_unit:
+            return AgeRange.parse(self.organism_age, self.organism_age_unit)
+        else:
+            return None
```

Comment:
Just curious, does this function take any value from `organism_age` and `organism_age_unit` and convert it to seconds, so that the data portal can display all ages in the same unit? Will age actually be displayed in seconds, or is seconds chosen as the most granular unit such that it can be converted to something more interpretable, like years?

Reply:
The intent is to facilitate queries, not necessarily the display. We might display the original metadata properties in the table but when it comes to filtering, the UI will probably display some sort of pair of range sliders for the donor age, with human-readable unit on it, and convert to seconds under the hood. The user will likely never have to deal with seconds.

Reply:
… unless the sequenced organism really was only  < 60 seconds old which I guess could happen for certain species.

Reply:
> unless the sequenced organism really was only < 60 seconds old

Highly unlikely :) Thanks for the explanation!